packages/workspace/test/workspace/workspace.test.ts

```diff
@@ -2128,10 +2130,19 @@ describe('workspace', () => {
       await expect(workspace.addEnvironment(envName, mockRmcToSource))
         .rejects.toEqual(new EnvDuplicationError(envName))
     })
-    it('should throw InvalidEnvNameError', async () => {
-      const invalidEnvName = 'invalid:env'
-      await expect(workspace.addEnvironment(invalidEnvName, mockRmcToSource))
-        .rejects.toEqual(new InvalidEnvNameError(invalidEnvName))
+
+    describe('invalid new environment name', () => {
+
+      test('throws InvalidEnvNameError for name with illegal characters', async () => {
+        const nameWithIllegalChars = 'invalid:env'
+        await expect(workspace.addEnvironment(nameWithIllegalChars, mockRmcToSource))
+            .rejects.toThrow(InvalidEnvNameError)
+      })
+      test('throws InvalidEnvNameError when name is too long', async () => {
+        const longName = new Array(MAX_ENV_NAME_LEN * 2).join('a')
```

Comment:
No need to use Array. There is a native string function that is probably more performant.
```suggestion
        const longName = '#'.repeat(MAX_ENV_NAME_LEN + 1)
```

Reply:
v